ocaml-lsp-server.opam

```diff
@@ -26,13 +24,20 @@ build: [
   [
     "dune"
     "build"
-    "-p"
-    name
     "-j"
     jobs
-    "@install"
-    "@runtest" {with-test}
-    "@doc" {with-doc}
+    "--root"
+    "."
+    "--ignore-promoted-rules"
+    "--no-config"
+    "--profile"
+    "release"
+    "%{name}%.install"
+  ]
+  [
+    "cp"
+    "%{_:build}%/_build/default/%{name}%.install"
+    "%{name}%.install"
```

Comment:
This is a sad hack to make it work both with dune 1.0 an dune 2.0 (in dune 2.0 `--ignore-promoted-rules` doesn't promote install files but `--promote-install-files` is only available in dune 2.0).

I think it's worth it.

Reply:
What is `%{_:build}%`?

Reply:
Build dir of the package opam is building currently.